app/views/common-content/prove-identity.html

```diff
@@ -42,11 +41,11 @@ <h2 class="heading-medium">{{formTitle}}</h2>
 					{% endif %}
 					{% if serviceString == 'parking-permit' and council.sixmonth %}
             <li>
-              if you've moved in the last 3 months you could only get a 3 month permit today
+              if you've moved in the last 3 months you may only get a 3 month permit today
             </li>
 					{% endif %}
             <li>
-              creating an account takes about 5 minutes and the rest of the application is around 2 minutes
+              create a Verify account in about 5 minutes, complete your application afterwards in around 2 minutes
```

Comment:
I think this might not be the right phrase – Isn't it an "identity account" or something @anthonylord?

Reply:
I can revert it to "creating an account". ["GOV.UK Verify account" should be correct (that's what it is, isn't it) but it's too long]

Reply:
I'm not sure it is a "GOV.UK Verify account" – Nobody creates an account with Verify, the account is with the IDP.

Reply:
IDP journey keeps the Verify logo though... and it's not a, for example, Barclays account, it's a Barclays Verify account
In any case this is a separate discussion, as I'll be reverting it to "create an account" for copy length